packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.scss

```diff
@@ -10,82 +10,68 @@
 
     @include mobile {
         padding-top: 0rem;
-        transition: height 0.2s ease-in;
         height: fit-content;
     }
 
-    &__close-button {
-        all: unset;
-        cursor: pointer;
-    }
-
     &__info {
         display: flex;
         justify-content: space-between;
         align-items: center;
-        padding-inline: 2.4rem;
+        padding-inline: 4rem;
 
         @include mobile {
             padding: 1.6rem 1.6rem 0rem;
         }
+    }
 
-        &__top-left {
-            display: flex;
-            flex-direction: column;
-            gap: 0.6rem;
-
-            &__details {
-                display: flex;
-                gap: 2.4rem;
-                align-items: center;
-
-                @include mobile {
-                    visibility: visible;
-                    transition: visibility 0s, height 0.2s ease;
-                    height: 2rem;
-                    gap: 0.8rem;
-                }
-
-                &__title {
-                    font-size: 1.6rem;
-
-                    @include mobile {
-                        font-size: 1.4rem;
-                    }
-                }
-            }
+    &__top-right-info {
+        display: flex;
+        gap: 1.6rem;
+        align-items: start;
 
-            &__balance {
-                font-size: 3.2rem;
-                font-weight: 700;
+        @include mobile {
+            gap: 0.8rem;
+        }
+    }
+
+    &__close-icon {
+        cursor: pointer;
 
-                @include mobile {
-                    font-size: 1.8rem;
-                }
+        &--white {
+            path {
+                fill: var(--light-8-primary-background, #fff);
             }
         }
+    }
 
-        &__top-right {
-            display: flex;
-            gap: 1.6rem;
-            align-items: start;
+    &__top-left-info {
+        display: flex;
+        flex-direction: column;
+        gap: 0.6rem;
+    }
 
-            @include mobile {
-                gap: 0.8rem;
-            }
+    &__details {
+        display: flex;
+        gap: 0.8rem;
+        align-items: center;
+        
+        @include mobile {
+            visibility: visible;
```

Comment:
is this line needed?

Reply:
@lubega-deriv It is done for the transition of hiding the wallet-details (name + badge) when the user scrolls on the wallets-cashier-content (if it is scrollable).

Reply:
@aum-deriv wouldn't it be visible by default? Could it not work if we only give the transition setting:
```
transition: visibility 0s, height 0.2s ease;
```